code/modules/research/designs/autolathe_desings/autolathe_designs_tools.dm

```diff
@@ -20,6 +20,14 @@
 	build_path = /obj/item/crowbar
 	category = list("initial","Tools")
 
+/datum/design/crowbar
+	name = "Large Crowbar"
+	id = "large_crowbar"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500)
+	build_path = /obj/item/crowbar/large
+	category = list("initial","Tools")
```

Comment:
I'm extremely apprehensive about this. I really don't want to see tool speeds, of all things, end up with powercreep as a result of a 0.5 toolspeed admin-only item getting added

Reply:
Oh then ill remove it